chain/docker-config/add-account.exp

```diff
@@ -2,11 +2,7 @@ set user [lindex $argv 0]
 set pw [lindex $argv 1]
 set mnemonic [lindex $argv 2]
 
-spawn bandcli keys add $user --recover
-expect "Enter a passphrase to encrypt your key to disk:"
-send "$pw\r"
-expect "Repeat the passphrase:"
-send "$pw\r"
+spawn bandcli keys add $user --recover --keyring-backend test
```

Comment:
Does this mean the passphrase is no longer needed? If so, then what if the user used a keystone from v0.37, would that be compatible with v0.38 here?

Reply:
They change it so that there are multiple ways you can store your key. If user wants to use the same one as 0.37, they pass `--keyring-backend file`. The default option now uses the machine's keychain store.